docs/test_related.md

```diff
@@ -0,0 +1,28 @@
+### Setup related Information 
```

Comment:
love this!  :) we should have added from the very beginning

Reply:
+1

Reply:
Suggestion: Rename file to `testing.md`

Reply:
fixed